src/uv-common.h

```diff
@@ -140,6 +140,11 @@ enum {
   UV_HANDLE_REAP                        = 0x10000000
 };
 
+#define uv__is_raw_tty_mode(m)                                                \
+  ((m) == UV_TTY_MODE_RAW ||                                                  \
+   (m) == UV_TTY_MODE_RAW_VT ||                                               \
+   (m) == UV_TTY_MODE_RAW_LEGACY)
```

Comment:
I'd rather have this as a `static inline` function so there's no risk of accidentally evaluating `m` more than once.

Reply:
Sure! Done in c785048904b99f42f873c861cf556aa592394d84